docs/baremetal-agent-controller.md

```diff
@@ -140,16 +130,20 @@ metadata:
 spec:
   online: true
   bootMACAddress: 00:ec:ee:f8:5a:ba
-  # Disable automated cleaning mode
   automatedCleaningMode: disabled
   bmc:
     address: ....
     credentialsName: bmc-secret
 ```
 
-The above is an optional step. If skipped, the `BareMetalHost` will boot IPA and spend some time in
-the inspecting phase when the manifest is applied. There's an [open PR](https://github.com/openshift-metal3/dev-scripts/pull/1193/)
-that will allow setting an environment variable to have inspection disabled by `dev-scripts` by default.
+Setting `automatedCleaningMode` field and the `inspect.metal3.io` annotation are both optional. If
+skipped, the `BareMetalHost` will boot IPA and spend some time in the inspecting phase when the
+manifest is applied. Setting the `infraenvs.agent-install.openshift.io` is required. Without it,
```

Comment:
I was suggest also stating  the value of "infraenvs.agent-install.openshift.io" label should be set to the name of the InfraEnv. The relationship between BMH and InfraEnv is not clear to a newbie. I think you go into some of it in the troubleshooting guide, but it may be worth explaining that in the beginning along with how ClusterDeployment and Agent is tied in.

Reply:
great suggestion, thanks